doc/Interns/Ty-Won/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+May 6th 2020
```

Comment:
forgot to put in docs.  PLease add the time to this document... so, Date AND time.  Thanks

Reply:
@markAtMicrosoft No problem! Done!